src/bin/ch-remote.rs

```diff
@@ -338,6 +339,11 @@ fn do_command(matches: &ArgMatches) -> Result<(), Error> {
         Some("counters") => {
             simple_api_command(&mut socket, "GET", "counters", None).map_err(Error::ApiClient)
         }
+        Some("vmm-ping") => {
+            let full_command = "vmm.ping";
+            simple_api_full_command_with_fds(&mut socket, "GET", &full_command, None, Vec::new())
+                .map_err(Error::ApiClient)
+        }
```

Comment:
Please simplify as follows:
```suggestion
        Some("ping") => {
            simple_api_full_command(&mut socket, "GET", "vmm.ping", None)
                .map_err(Error::ApiClient)
        }
```